test/22_os-config.ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from 'chai';
+
+import { supertest } from './test-lib/supertest';
+import { LOGS_HOST } from '../src/lib/config';
+
+describe('OS configuration endpoints', () => {
+	describe('/os/v1/config', () => {
+		it('should return a valid JSON response', async () => {
+			const { body } = await supertest().get('/os/v1/config').expect(200);
+
+			expect(body)
+				.to.have.property('services')
+				.that.has.all.keys('openvpn', 'ssh');
```

Comment:
How about moving here the respective test case of the bC-api?
See: https://github.com/balena-io/balena-api/blob/master/test/27_os.ts#L10-L11